### PR TITLE
adding linters to pure_pursuit package

### PR DESCRIPTION
**Pull request by nik-tier4**

### CODE REVIEW

File: control/pure_pursuit/CMakeLists.txt [hunk 1]
```diff
@@ -3,6 +3,8 @@ project(pure_pursuit)
 
 if(NOT CMAKE_CXX_STANDARD)
 	set(CMAKE_CXX_STANDARD 14)
+        set(CMAKE_CXX_STANDARD_REQUIRED ON)
+        set(CMAKE_CXX_EXTENSIONS OFF)
```
Comment (jilaada) [inline control/pure_pursuit/CMakeLists.txt:7]: Could you please fix the indenting here?

**Review by jilaada — COMMENTED** [review]
Looks like most of the changes here were from uncrustify so LGTM, one comment:

File: control/pure_pursuit/CMakeLists.txt (hunk 1 above)
Comment (nik-tier4) [reply control/pure_pursuit/CMakeLists.txt]: fixed!

**Review by jilaada — APPROVED** [review]
LGTM! Tested on my local system

**Pull request merged**